matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/auth/login/DefaultLoginWizard.kt

```diff
@@ -100,9 +100,7 @@ internal class DefaultLoginWizard(
 
     override suspend fun loginCustom(data: JsonDict): Session {
         val loginParams = data.toMutableMap()
-        if (matrixConfiguration.enableRefreshTokenAuth) {
-            loginParams["refresh_token"] = true
-        }
+        loginParams["refresh_token"] = enableRefreshTokenAuth
```

Comment:
This is fine, but not strictly equivalent to the previous version where the value was not set to the Json body when equal to `false`. Do you confirm?

Reply:
👍 Yea I changed it to have that behaviour to be consistent with the other login api's where the parameter is always present.